examples/demo_vrf.pp

```diff
@@ -16,9 +16,27 @@
 
 class ciscopuppet::demo_vrf {
   cisco_vrf { 'test_vrf':
-    ensure      => present,
-    description => 'test vrf for puppet',
-    shutdown    => false,
-    vrf         => 4096,
+    ensure              => present,
+    description         => 'test vrf for puppet',
+    # route_distinguisher is not supported on all platforms
```

Comment:
Add platform selector instead of commenting this out completely.

Here is an example: https://github.com/cisco/cisco-network-puppet-module/blob/develop/examples/demo_portchannel.pp#L24